doc/sphinx/addendum/implicit-coercions.rst

```diff
@@ -152,12 +135,16 @@ Declaring Coercions
   .. exn:: Cannot recognize @class as a source class of @qualid.
      :undocumented:
 
-  .. warn:: @qualid does not respect the uniform inheritance condition.
-     :undocumented:
-
   .. exn:: Found target class ... instead of ...
      :undocumented:
 
+  .. warn:: @qualid does not respect the uniform inheritance condition.
+
+     The :ref:`test for ambiguous coercion paths <ambiguous-paths>`
```

Comment:
Not necessary to hyperlink to a point so close the hyperlink (only 3 lines in the rst file).

But as I suggested, ambiguous coercion paths should be described in the main text and this hyperlink should point to there.